public/app/features/alerting/unified/utils/rule-form.ts

```diff
@@ -118,6 +121,10 @@ export const getDefautManualRouting = () => {
 
 export function formValuesToRulerRuleDTO(values: RuleFormValues): RulerRuleDTO {
   const { name, expression, forTime, forTimeUnit, keepFiringForTime, keepFiringForTimeUnit, type } = values;
+
+  const annotations = arrayToRecord(cleanKeyValuePairs(values.annotations));
```

Comment:
Empty keys are not supported in annotations and labels?

Reply:
Hm they should, let me make sure we still do

Reply:
Fixed :D